kubernetes-skills/claude/k8s-kind/SKILL.md

```
@@ -0,0 +1,262 @@
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/latest/kind-linux-amd64
+chmod +x ./kind
+sudo mv ./kind /usr/local/bin/kind
+
+# Verify
+kind version
+```
+
+## kind vs vind (vCluster)
+
+| Feature | kind | vind (vCluster) |
+|---------|------|-----------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)